src/components/blocks/dividers/Aside.tsx

```diff
@@ -17,11 +17,15 @@ import {
 
 const Aside = ({ data, attribs }: HtmlComponentProps<'div'>) => {
   const isVersioningInfo: boolean =
-    attribs && (attribs[`data-type`] === 'new' || attribs[`data-type`] === 'updated') ? true : false;
+    attribs &&
+    (attribs[`data-type`] === 'new' || attribs[`data-type`] === 'updated' || attribs[`data-type`] === 'experimental')
+      ? true
+      : false;
```

Comment:
We don't need the ternary conditional pointing to true/false here, something like 
```
attribs && (attribs[`data-type`] === 'new' || attribs[`data-type`] === 'updated' || attribs[`data-type`] === 'experimental')
```
or more concisely
```
Object.keys(versioningColors).includes(attribs?.[`data-type`] ?? '')
```
would do the same